apps/modernization-ui/src/apps/patient/profile/summary/PatientProfileSummary.tsx

```diff
@@ -88,7 +88,7 @@ export const PatientProfileSummary = ({ patient }: Props) => {
             ) : (
                 <>
                     <div className="border-bottom border-base-lighter patient-summary-title">
-                        <h2>{summary?.legalName && displayName('fullLastFirst')(summary.legalName)}</h2>
+                        <h2>{(summary?.legalName && displayName('fullLastFirst')(summary.legalName)) || '-- -- --'}</h2>
```

Comment:
Question: Should there just be a different option within `displayName` to return `-- -- --` in the case of no data?

Reply:
1. I would not think so because the `displayName`  is used in patient search as well we don't want the `-- -- -- ` to reflect there it would only be for patient profile
2. Yes let me change that.